template.rb

```diff
@@ -135,6 +133,29 @@ class Rack::Attack
 
 run 'cp config/environments/production.rb config/environments/staging.rb'
 
+file '.travis.yml', <<-RUBY
+language: ruby
+rvm:
+  - #{RUBY_VERSION}
+before_script:
+  - createdb #{app_name}_test
+  - cp config/database.yml{.travis,}
+  - cp config/secrets.yml{.travis,}
+  - 'bundle exec bundle-audit update && bundle exec bundle-audit check'
+cache: bundler
+addons:
+  code_climate:
+    repo_token: <add here>
+notifications:
+  slack:
+    on_failure: always
+    on_success: change
+    rooms:
+      - rawnet: <add here>
```

Comment:
First part is always the same I think?
`rawnet:uT8j6gbU1RpxOVmgozTh7paX#<add here>`